integration-tests/overlays/cnv-prod-ocp4-stage/job-generate-name.yaml

```diff
@@ -0,0 +1,6 @@
+- op: move
+  from: /metadata/name
+  path: /metadata/generateName
+- op: add
+  path: /metadata/namespace
+  value: "thoth-frontend-stage"
```

Comment:
I've put it into frontend namespace - infra namespace required patching also imagestream if we want to run different versions of integration tests for stage/prod.